Makefile

```diff
@@ -0,0 +1,47 @@
+PYTHON = python3
+
+VENV      = venv
+ACTIVATE := source $(VENV)/bin/activate
```

Comment:
This does not seem to work on my system. Changing `source` to `.` (dot) solves the issue.

Reply:
I'm not sure I understand what you're suggesting

You can make a proper inline suggestion...

Reply:
```suggestion
ACTIVATE := . $(VENV)/bin/activate
```

Reply:
I did not really understand why that helps but it might be related to the fact that make files execute statements in `sh` and that might not support `source` on all platforms (e.g. Ubuntu).

Reply:
Tried it and it works. thanks!